toolkit/package.json

```diff
@@ -71,7 +71,7 @@
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^5.1.0",
     "prettier": "^3.1.1",
-    "rollup": "^4.9.1",
+    "rollup": "^3.29.4",
```

Comment:
Saw your note about not upgrading rollup but just wanted to check if this is an intentional downgrade?

Reply:
Sorry - from a commit view. Again, I see this isn't relevant in full diff